src/pages/auth/Login.jsx

```diff
@@ -77,16 +59,17 @@ return <>
             </div>
 
             <div className="form-btn ">
-                <button className="btn login padd-sm" type="submit">Login</button>
+                <button className="btn login padd-sm margin-btm" type="submit">Login</button>
+            </div>
+            <div className="form-btn ">
+                <button className="nav-login-btn" onClick={(e)=>{ e.preventDefault(); loginSubmitHandler(e,guestUser) } }>Login as guest</button>
```

Comment:
You can remove `e.preventDefault()` here since you're re-invoking it inside `loginSubmitHandler` method

Reply:
ooh forgot , thank you